internal/backend/sessions/sessionworkflows/workflows.go

```diff
@@ -234,6 +234,9 @@ func (ws *workflows) sessionWorkflow(wctx workflow.Context, params *sessionWorkf
 				err = nil
 			} else {
 				l.Sugar().Errorf("session workflow error: %v", err)
+				if errors.As(err, &sdkerrors.RetryableErrorType) {
+					panic("panic session to retry")
```

Comment:
should be `l.Panic(...)`

Reply:
i also wouldn't count it as a session error for the statistics. i would make it have another stat.

Reply:
1. Done